webapp/TargetedMS/js/QCPlotHelperBase.js

```diff
@@ -688,6 +691,33 @@ Ext4.define("LABKEY.targetedms.QCPlotHelperBase", {
         this.attachPlotExportIcons(id, mainTitle + '-' + this.precursors[precursorIndex] + '-' + this.getMetricPropsById(this.metric).series1Label, plotIndex, this.getPlotWidth(), extraMargin);
     },
 
+    getYAxisPosition: function(precursorInfo, leftMargin)
+    {
+        var offset = 15;
+
+        return this.getYAxisLeftMarginOffset(precursorInfo) > 0 ?
+                (leftMargin + this.getYAxisLeftMarginOffset(precursorInfo)) - offset :
+                undefined
+    },
+
+    getYAxisLeftMarginOffset: function(precursorInfo)
+    {
+        var maxLength = precursorInfo.min.toString().length;
+
+        if (maxLength < precursorInfo.max.toString().length) {
+            maxLength = precursorInfo.max.toString().length;
+        }
```

Comment:
I think this can be simplified to a single line using `Math.max()`
```suggestion
        var maxLength = Math.max(precursorInfo.min.toString().length, precursorInfo.max.toString().length)
```

Reply:
Also, I assume there is no reason that the`precursorInfo.min` or `precursorInfo.max` would ever be undefined, but might be worth a quick check.
```
if (precursorInfo.min === undefined || precursorInfo.max === undefined) {
    return 0;
}  
```

Reply:
done